Platform/ElkhartlakeBoardPkg/BoardConfig.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, *args, **kwargs):
         self.FLASH_BASE_SIZE      = (self.FLASH_LAYOUT_START - self.FLASH_BASE_ADDRESS)
 
         self.HAVE_FIT_TABLE       = 1
-        self.HAVE_FSP_BIN         = 0
+        self.HAVE_FSP_BIN         = 1

```

Comment:
If a FSP is ready, can you do this in a separate PR?

Reply:
yeah this is a mistake. other than this, do you think the basic flow/design is fine?